package.json

```diff
@@ -82,6 +82,7 @@
     "yeoman-generator": "^1.0.1"
   },
   "devDependencies": {
+    "@polymer/tools-common": "^1.0.1",
```

Comment:
IMO we should just be 100% pure semver here. As it's a team internal package there is not even an optics concern to rolling out however many breaking changes we need